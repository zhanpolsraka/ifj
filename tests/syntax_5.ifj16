//2
class Test{
    //Chybi typ u parametru b
    static int Sum(int a, b) {
        return a+b;
    }
}
class Main{
    static void run() {
        int a = 0;
        int b = 1;
        int sum = Test.Sum(a, b);
    }
}
