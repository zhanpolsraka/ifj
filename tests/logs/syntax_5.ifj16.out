
Syntax error! Unknown expression with token "b"!

